### DISCUSSION TITLE
jetson-nano-devkit-emmc not found

### DISCUSSION THREAD

**Discussion by danny1209** [OE4T/tegra-demo-distro#235] [Q&A] [answered]
I downloaded the dunfell repo. 
. ./setup-env --machine jetson-nano-devkit-emmc

then I get this: 
![image](https://user-images.githubusercontent.com/46848895/204001488-663d69c9-d91e-4420-af50-4c634a3c26f5.png)

As I know a few months ago it works, why now not found?

**Comment by ichergui** [comment]
Hey @danny1209 
Did you follow the instructions described in the `README` ?
It is working fine

**Comment by dwalkes** [comment, marked as answer]
I suspect you may not be on the dunfell branch since [this branch doesn't include agx-orin](https://github.com/OE4T/meta-tegra/tree/dunfell/conf/machine) as shown in your screenshot.  You are probably on the master branch, which doesn't include nano devkit anymore since nano isn't supported in Jetpack 5.

**Reply by danny1209** [reply]
oh yes, I made a mistake! thanks a lot!
